doc/manual.rst

```diff
@@ -140,10 +140,20 @@ statistics can be printed for the inspection.
 >>> automl = autosklearn.classification.AutoSklearnClassifier()
 >>> automl.fit(X_train, y_train)
 >>> automl.cv_results_
+>>> automl.performance_over_time_.plot(
+>>>    x='Timestamp',
+>>>    kind='line',
+>>>    legend=True,
+>>>    title='Auto-sklearn accuracy over time',
+>>>    grid=True,
+>>> )
+>>> plt.show()
+>>> 
 >>> automl.sprint_statistics()
 >>> automl.show_models()
 
 ``cv_results_`` returns a dict with keys as column headers and values as columns, that can be imported into a pandas DataFrame.
+``performance_over_time_``  returns a DataFrame containing the models performance over time data, which can be used for plotting directly (`Train and Test inputs <examples/40_advanced/example_pandas_train_test.html>`_).
 ``sprint_statistics()`` is a method that prints the name of the  dataset, the metric used, and the best validation score
```

Comment:
Sorry I should have spotted this one earlier, the `linkcheck` failed in the last test. This should be changed to :ref:`sphx_glr_examples_40_advanced_example_pandas_train_test.py` instead of the html link.

In general with sphinx, I would reserve html links only for external links and everything internal to use ref:\`My text \<link\>\`

i.e.
```rst
:ref`Here's an example <sphx_glr_examples_40_advanced_example_pandas_train_test.py>`
```

You can check the `linkcheck` command that's present in `doc/Makefile` by using:
```bash
cd doc
make linkcheck
```

This will automatically cause the examples to run again so as a quick workaround you can modify the `linkcheck` command locally to prevent rerunning the examples. They take a while to run and are not important to identifying if the link works or not. **Don't commit these changes, it's just for your own local check to make sure it works**
```
# In Makefile
...
# Old
linkcheck:
	$(SPHINXBUILD) -b linkcheck $(ALLSPHINXOPTS) $(BUILDDIR)/linkcheck
# New
linkcheck:
	$(SPHINXBUILD) -D plot_gallery=0 -b linkcheck $(ALLSPHINXOPTS) $(BUILDDIR)/linkcheck
```
`sphinx-gallery` is used to run the examples and generate the outputs and convert it all to html. The `-D plot_gallery=0` part just tells `sphinx` not to run `sphinx-gallery`, it just parses the text but doesn't actually run the examples with Python.

*(we should probably have this flexible kind of command in anyways)*